src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -270,43 +227,63 @@ page 2500 "Extension Management"
                     end;
 #endif
                 }
+                action("Microsoft AppSource Gallery")
+                {
+                    Caption = 'AppSource Gallery';
+                    Enabled = IsSaaS;
+                    Image = NewItem;
+                    ToolTip = 'Browse the Microsoft AppSource Gallery for new extensions to install.';
+                    Visible = not IsOnPremDisplay;
+                    RunObject = page "AppSource Product List";
+                    RunPageMode = View;
+                }
                 action("Upload Extension")
                 {
-                    ApplicationArea = All;
                     Caption = 'Upload Extension';
                     Image = Import;
-                    Promoted = true;
-                    PromotedOnly = true;
-                    PromotedCategory = Category5;
-                    PromotedIsBig = true;
                     RunObject = page "Upload And Deploy Extension";
                     ToolTip = 'Upload an extension to your application.';
                     Ellipsis = true;
                     Visible = IsSaaS;
                 }
                 action("Deployment Status")
                 {
-                    ApplicationArea = All;
                     Caption = 'Installation Status';
                     Image = Status;
-                    Promoted = true;
-                    PromotedOnly = true;
-                    PromotedCategory = Category5;
-                    PromotedIsBig = true;
                     RunObject = page "Extension Deployment Status";
                     ToolTip = 'Check status for upload process for extensions.';
                     Visible = IsSaaS;
                 }
                 action("Delete Orphaned Extension Data")
                 {
-                    ApplicationArea = All;
+
                     Caption = 'Delete Orphaned Extension Data';
                     Image = Delete;
                     RunObject = page "Delete Orphaned Extension Data";
                     ToolTip = 'Delete the data of orphaned extensions.';
                 }
             }
         }
+        area(Promoted)
+        {
+            group(Category_Category5)
+            {
+                Caption = 'Manage', Comment = 'Generated from the PromotedActionCategories property index 4.';
```

Comment:
I have no idea what this comment is trying to tell me...

Reply:
To address in this PR